cluster/jmx-client/pekko-cluster

```diff
@@ -46,16 +46,14 @@ function ensureNodeIsRunningAndAvailable {
     REPLY=$(get Available 2>&1) # redirects STDERR to STDOUT before capturing it
     if [[ "$REPLY" != *true ]]; then
         if [[ "$REPLY" == *"Cannot convert result to a string." ]]; then
-          echo "Akka cluster MBean is not available on $HOST:$PORT with MBean name '$(mbeanObjectName)'"
+          echo "Pekko cluster MBean is not available on $HOST:$PORT with MBean name '$(mbeanObjectName)'"
         else
-          echo "Akka cluster node is not available on $HOST:$PORT with MBean name '$(mbeanObjectName)', due to $REPLY"
+          echo "Pekko cluster node is not available on $HOST:$PORT with MBean name '$(mbeanObjectName)', due to $REPLY"
         fi
         exit 1
     fi
 }
 
-echo "This jmx-client/akka-cluster tool is deprecated use curl and https://github.com/akka/akka-cluster-management instead, since 2.5.0." >&2
```

Comment:
could you put this `echo` back but replace the URL with https://github.com/apache/incubator-pekko-management? 
- and remove the `, since 2.5.0` bit